app/page.tsx

```diff
@@ -5,8 +5,8 @@ import React from "react";
 
 export default function Home() {
   return (
-    <div className="flex justify-center items-start h-screen p-4">
-      <Card isBlurred className="mt-24 max-w-md p-4 border border-stone-300" shadow="lg">
```

Comment:
When I removed h-screen I changed this to be 16 to compensate for the removed div size